src/igvfd/searches/configs/file_set.py

```diff
@@ -43,13 +67,37 @@ def file_set():
             'audit.INTERNAL_ACTION.category': {
                 'title': 'Audit Category: Internal Action'
             },
+            'type': {
+                'title': 'Object Type',
+            },
         },
         'facet_groups': [
             {
                 'title': 'File Set',
                 'facet_fields': [
                     'file_set_type',
+                    'assembly',
+                    'transcriptome_annotation',
+                ],
+            },
+            {
+                'title': 'Sample',
+                'facet_fields': [
                     'donors.taxa',
+                    'samples.classifications',
+                    'samples.sample_terms.term_name',
+                    'samples.targeted_sample_term.term_name',
+                    'samples.disease_terms.term_name',
+                    'samples.modifications.modality',
+                    'samples.treatments.treatment_term_name',
+                    'sequencing_library_types',
```

Comment:
`sequencing_library_types` not in the `facets` object.

Reply:
done